docs/index.md

```diff
@@ -2,6 +2,16 @@
 
 ![Architecture](images/oasis-core-high-level.svg)
 
+## Development Setup Examples
```

Comment:
I don't like the "Examples" in the title since the links below actually include our canonical prerequisites and building instructions.

Perhaps we should split these links into 2 sections, e.g.:
- "Build Environment Setup and Building"
  - [Prerequisites]
  - [Building]
- "Running Tests and Development Networks"
  - [Running Tests]
  - [Local Network Runner With a Simple Runtime]
  - [Single Validator Node Network]

Reply:
Good idea, done.